src/Pos/Launcher/Param.hs

```diff
@@ -61,7 +62,7 @@ data NodeParams = NodeParams
     , npSecretKey      :: !SecretKey            -- ^ Primary secret key of node
     , npUserSecret     :: !UserSecret           -- ^ All node secret keys
     , npBaseParams     :: !BaseParams           -- ^ See 'BaseParams'
-    , npGenesisUtxo    :: !GenesisUtxo          -- ^ Predefined genesis utxo
+    , npGenesisTxpCont :: !GenesisTxpContext    -- ^ Predefined genesis context related to txp data.
```

Comment:
We use `ctx` as a short version of `context`, don't use `cont`, it's not conventional.